harness/determined/pytorch/dsat/__init__.py

```diff
@@ -13,6 +13,7 @@
     DSATModelProfileInfoTrial,
     RandomDSATSearchMethod,
     BinarySearchDSATSearchMethod,
+    ASHADSATSearchMethod,
```

Comment:
mypy behaves without putting this in `__all__`? That's good to know, perhaps it isn't necessary

Reply:
Uh, I didn't see any issues in the IDE without putting it in `__all__`, but I'll add it there to be safe